src/index.html

```diff
@@ -9,6 +9,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#image">
+        <img
+        class="image"
+        src="images\moyo_1.png"
+        alt="logo"
+        >
+      </a>
+      <nav class="nav">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav">
```

You should add one empty line between each tag of one parent and if that tag takes more then one line